src/Symfony/Component/Dotenv/Dotenv.php

```diff
@@ -65,7 +65,7 @@ public function load($path, ...$paths)
      *
      * @param array $values An array of env variables
```

Comment:
If PHP7.1, then `iterable` is a solution. But it depends, If you really want an array, it's not the same as requiring an iterable. For example, if you need to **count** the content, you can't rely on `Traversable`. So it really depends on the situation